README.md

```diff
@@ -33,8 +36,19 @@ When you have cloned the repo you can then execute `EnvironmentSetup.sh` which w
 $ cd ~/Programming/robotics-prototype
 $ ./EnvironmentSetup.sh
 ```
+After you have restarted your terminal you can run the GUI
+
+### Running the GUI
+1. Make sure your virtual environment is activated.
+#### Option 1
+2. Use the `base` alias to enter the necessary directory.
+3. Execute `app.py` using `python app.py`
```

Comment:
You're not running ROS. you gotta deactivate venv, `roslaunch rosbridge_server websocket...` to get `roscore` and the bridge between ROS and JS running. I would really just put the rosgui/startgui steps. Also since people might be unfamiliar with linux explicit say that they have to open a new terminal. (Some guy once wrote rosgui CTRL+C'ed it and then put startgui)

Reply:
Foq